app/views/checklists/show.html.erb

```diff
@@ -49,35 +51,22 @@
     ]
     @tabsets = { right: draw_tab_set(tabs) }
   end
+
+  @container = :full
+  names = @data.species
 %>
 
-<div class="text-center">
+<div class="my-4">
   <%= :checklist_summary.t(species: @data.num_species,
                            genera: @data.num_genera) %>
-  <br/>
-  <br/>
-
-  <style>
-    table.checklist td {
-      padding: 0px 1em 0px 1em;
-    }
-  </style>
+</div>
 
-  <table class="checklist">
-    <% num_cols = 5
-    names = @data.species
-    col_length = ((names.length + num_cols - 1).to_f / num_cols).truncate
-    (1..col_length).each do |row| %>
-      <tr>
-        <% (1..num_cols).each do |col| %>
-          <% i = (col-1) * col_length + row - 1
-          if i < names.length %>
-            <td><i>
-              <%= link_to(names[i], name_link.merge(name: names[i])) %>
-            </i></td>
-          <% end %>
-        <% end %>
-      </tr>
-    <% end %>
-  </table>
-</div><!--.text-center-->
+<div class="panel panel-default">
+  <div class="panel-body checklist">
+    <ul class="list-unstyled">
+      <% names.each do |name| %>
+        <li><i><%= link_to(name, name_link.merge(name: name)) %></i></li>
```

Comment:
So, so throwing this out there, but if this is the only reason it has to look up every single name in the list... why not use the lookup_name link instead?  That defers the lookup until after you click on the name, right?

Reply:
(I mean, obviously, a deeper refactor could potentially solve the problem much better, and all!  But this could be a super-simple way of solving the huge N+1 problem in the short term!)